Cargo.toml

```diff
@@ -25,27 +25,31 @@ hyper-v = ["winapi", "widestring", "ntapi", "vid-sys"]
 
 
 [dependencies]
-log = "0.4.8"
-env_logger = "0.7.1"
-libc = { version = "0.2.58", optional = true }
+log = "0.4"
+env_logger = "0.8"
+libc = { version = "0.2", optional = true }
 xenctrl = { version = "0.4.2", optional = true }
 xenstore-rs = { version = "0.3.0", optional = true }
 xenforeignmemory = { version = "0.1.0", optional = true }
 xenevtchn = { version = "0.1.2", optional = true }
 xenvmevent-sys = { version = "0.1.3", optional = true }
 kvmi = { version = "0.2.1", optional = true }
 fdp = { version = "0.1.0", optional = true }
-winapi = { version = "0.3.8", features = ["tlhelp32", "winnt", "handleapi", "securitybaseapi"], optional = true }
-widestring = { version = "0.4.0", optional = true }
-ntapi = { version = "0.3.3", optional = true }
+winapi = { version = "0.3", features = ["tlhelp32", "winnt", "handleapi", "securitybaseapi"], optional = true }
+widestring = { version = "0.4", optional = true }
+ntapi = { version = "0.3", optional = true }
 vid-sys = { version = "0.3.0", features = ["deprecated-apis"], optional = true }
 bitflags = "1.2.1"
 cty = "0.2.1"
-nix = "0.18.0"
+nix = "0.19"
 
 [dev-dependencies]
-ctrlc = "3.1.3"
-clap = "2.33.0"
-colored = "1.9.3"
-mockall = "0.7.1"
+ctrlc = "3.1"
+clap = "2.33"
+colored = "2.0"
+mockall = "0.9"
 test-case = "1.0.0"
+
+# Wait for test-case 1.0.1 release
+[patch.crates-io]
```

Comment:
let's wait for the official release on crates.io,
I don't want to merge a patched crate based on a git repo.

Reply:
Sure, but it's uncertain when that will be and until then the CI is broken. You could try to pin `syn` to an older version instead of patching `test-case`, but I tried and I was not successful with that.

Reply:
Then just pin it to a specific commit with `ver`.
any changes on test-case master branch can brake our CI with this.

Reply:
Edit: sorry `rev`, not `ver`

Reply:
Never mind, turns out they were quick about the release. ;)